docs/cli/build.md

```diff
@@ -39,11 +39,12 @@ faas-cli build --lang python3 --build-option dev [--build-option debug]
 
 or in YAML:
 
-```yaml
-    build_options:
-    - debug
-    - dev
-```
+!!! example "stack.yml"
```

Comment:
New one on me. Is the indentation needed?

Reply:
Apparently.  Additionally it seems I may not have indented enough.

Reply:
Merge or no?

I'm not keen on additional maintenance